sbom/examples/product/rhel-99.9.spdx.json

```diff
@@ -0,0 +1,120 @@
+  "packages": [
+    {
+      "SPDXID": "SPDXRef-RHEL-99.9",
+      "name": "Red Hat Enterprise Linux",
+      "versionInfo": "99.9",
+      "supplier": "Organization: Red Hat",
+      "downloadLocation": "https://developers.redhat.com/products/rhel/download",
+      "licenseConcluded": "NOASSERTION",
+      "externalRefs": [
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/o:redhat:enterprise_linux:10::baseos",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::appstream",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::buildroot",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::crb",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::highavailability",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::nfv",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::realtime",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::resilientstorage",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::sap",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::sap_hana",
+          "referenceType": "cpe22Type"
+        },
+        {
+          "referenceCategory": "SECURITY",
+          "referenceLocator": "cpe:/a:redhat:enterprise_linux:10::supplementary",
+          "referenceType": "cpe22Type"
+        }
+      ]
+    },
+    {
+      "SPDXID": "SPDXRef-SRPM-1",
+      "name": "openssl",
+      "versionInfo": "100.0.0-10.el99_9",
+      "supplier": "Organization: Red Hat",
+      "downloadLocation": "NOASSERTION",
+      "packageFileName": "openssl-100.0.0-10.el99_9.src.rpm",
+      "licenseConcluded": "Apache-2.0",
+      "externalRefs": [
+        {
+          "referenceCategory": "PACKAGE-MANAGER",
+          "referenceType": "purl",
+          "referenceLocator": "pkg:rpm/redhat/openssl@100.0.0-10.el99_9?arch=src"
+        },
+        {
+          "referenceCategory": "OTHER",
+          "referenceType": "sbom_ref",
```

Comment:
Do you know `sbom_ref` is defined? I'm having trouble finding a list of valid reference types.

I'm wondering what flexibility we have with the referenceLocator.

Reply:
It is not defined, see also https://github.com/RedHatProductSecurity/security-data-guidelines/pull/3#discussion_r1640153136. I'll rework this example to use DocumentRefs and when we add CycloneDX examples in the future, they'll use a `"type": "bom"` reference type.

Reply:
In the end, the decision is to refer to other objects purely with purls. DocumentRefs are not widely adopted by the industry yet, so we can rely on purls for now only. We can always add more explicit references later on once the tooling catches up.